kapten/tool.py

```diff
@@ -92,7 +92,11 @@ async def list_services(self, image: Optional[str] = None) -> List[Service]:
 
         # Assert we got the services we asked for
         if len(services) != len(self.service_names):
-            raise KaptenError("Could not find all tracked services")
+            found_services = {s.name for s in services}
+            missing = ", ".join(sorted(set(self.service_names) - found_services))
+            raise KaptenError(
+                f"Could not find all tracked services. Missing: {missing}"
+            )
```

Comment:
We could do:

```python
missing_services = set(self.service_names) - {s.name for s in services}
if missing_services:
    missing = ", ".join(sorted(missing))
    raise KaptenError(f"Could not find all tracked services. Missing: {missing}")
```

Though that logically changes stuff and I'm not really sure if that would have any implications.

Reply:
I think this change would be safe. Maybe @lundberg can confirm?